docs/source/quickstart.md

```diff
@@ -2,6 +2,14 @@
 
 # Quickstart
 
+This tutorial covers the basic configuration and functionality of OpenNMT-py and makes sure that everything is up and running for Linux based systems. Do not forget to install your Nvidia graphic card's latest drivers and CUDA before applying the steps below. 
+
+```bash
+nvidia-smi
+```
+
+Use [Google Collab](https://colab.research.google.com/drive/1YzHT4av2SPXI_CzpX3mF9ImFv-V39n4D?usp=sharing) to check every steps quickly without any installation on your local machine. 
```

Comment:
This is not very clear. Maybe explicitly tell that you can either: 1. run on your own GPU 2. run on colab if needed.